internal/protoplugin/protoplugin.go

```diff
@@ -97,8 +97,9 @@ func NewRunner(
 	templateInfoChecker func(*TemplateInfo) error,
 	baseImports []string,
 	fileToOutputFilename func(*File) (string, error),
+	unknownFlagHandler func(key string, value string) error,
```

Comment:
optional future change: parameter struct would be nice here

(this is internal so i don't care as much.)